### PR TITLE
Bugfix/visual mode markers

### PR DESCRIPTION
**Pull request by timtmok**
### Intent
Address #10949 and #10483

### Approach
Various fixes to visual mode markers. Some were issues with the virtual gutter because disabling line numbers means ACE hides the gutter. We have to create our own "gutter" to display the markers.

* lint items without the raw text caused blank tooltips (can't use the lint text because it has HTML tags)
* visual mode chunks didn't process _style_ type items so those markers didn't have the correct CSS classes applied
* a visual mode chunk at the beginning of the file wasn't initialized properly (no scope meant no way to tell if the lint item belonged to the chunk)
* appended lint text to gutter marker tooltips

### Automated Tests
This should fix an existing automated test

### QA Notes
* use the sample Rmd code in the issues
* disable visual mode line numbers (Options > R Markdown > Visual > Show line numbers in code blocks
* viewing tooltips requires the cursor to be in the code chunk (ACE limitation?)

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunkRowState.java [hunk 1]
```diff
@@ -136,7 +136,15 @@ protected void removeClazz()
    @Override
    protected void setTitle(String title)
```
Comment (kevinushey) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunkRowState.java:137]: This change feels a little strange. Should we instead have a function like `appendToTitle()` for the cases where we want to append content?

Comment (kevinushey) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunkRowState.java:137]: Or instead, the caller of `setTitle()` should be querying the current title and appending to it instead; I don't think that should be happening here.

Comment (timtmok) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunkRowState.java:137]: I like the suggestion of adding `appendToTitle()`

**Review by kevinushey — APPROVED** [review]
LGTM!

**Pull request merged**